src/cityreader.py

```diff
@@ -16,13 +24,18 @@
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
 
+import csv
```

Comment:
You need to name the csv file here. When I run cityreader, it doesn't display anything because it's not receiving the cities.csv file.